[2022-02-05 19:06:02,535] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2019-03-02T06:00:00+00:00 [queued]>
[2022-02-05 19:06:02,577] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2019-03-02T06:00:00+00:00 [queued]>
[2022-02-05 19:06:02,578] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-05 19:06:02,581] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-05 19:06:02,583] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-05 19:06:02,610] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-03-02 06:00:00+00:00
[2022-02-05 19:06:02,621] {standard_task_runner.py:52} INFO - Started process 211 to run task
[2022-02-05 19:06:02,637] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'download_dataset_task', 'scheduled__2019-03-02T06:00:00+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dags_hw.py', '--cfg-path', '/tmp/tmp6q7xlhhh', '--error-file', '/tmp/tmplmtgmlpc']
[2022-02-05 19:06:02,639] {standard_task_runner.py:77} INFO - Job 93: Subtask download_dataset_task
[2022-02-05 19:06:02,764] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag.download_dataset_task scheduled__2019-03-02T06:00:00+00:00 [running]> on host 6aa22590a839
[2022-02-05 19:06:02,853] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-05 19:06:02,964] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-03-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-03-02T06:00:00+00:00
[2022-02-05 19:06:02,981] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-05 19:06:02,999] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-03.csv > /opt/***/yellow_tripdata_2019-03.csv']
[2022-02-05 19:06:03,088] {subprocess.py:85} INFO - Output:
[2022-02-05 19:12:59,835] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-05 19:13:36,921] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag, task_id=download_dataset_task, execution_date=20190302T060000, start_date=20220205T190602, end_date=20220205T191336
[2022-02-05 19:13:46,003] {local_task_job.py:212} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-02-05 19:13:50,025] {process_utils.py:124} INFO - Sending Signals.SIGTERM to group 211. PIDs of all processes in the group: [211]
[2022-02-05 19:13:50,130] {process_utils.py:75} INFO - Sending the signal Signals.SIGTERM to group 211
[2022-02-05 19:13:57,872] {process_utils.py:70} INFO - Process psutil.Process(pid=211, status='terminated', exitcode=0, started='19:06:02') (211) terminated with exit code 0
